ompi/mpi/c/intercomm_create.c

```diff
@@ -80,6 +81,20 @@ int MPI_Intercomm_create(MPI_Comm local_comm, int local_leader,
         */
     }
 
+#if OPAL_ENABLE_FT_MPI
+    /*
+     * An early check, so as to return early if we are using a broken
+     * communicator. This is not absolutely necessary since we will
+     * check for this, and other, error conditions during the operation.
+     */
+    if( OPAL_UNLIKELY(!ompi_comm_iface_create_check(local_comm, &rc)) ) {
```

Comment:
This is not good. If the bridge communicator is FT marked on both sides this will work, but a side cannot leave earlier and certainly not based on a local decision (that will not be known by the remote side).